### PR TITLE
feat: Add FeeHook and Swell to ezEth

### PR DESCRIPTION
**Pull request by ltyu**

### CODE REVIEW

File: typescript/infra/config/environments/mainnet3/warp/configGetters/getRenzoEZETHWarpConfig.ts [hunk 1]
```diff
@@ -23,7 +26,12 @@ const chainsToDeploy = [
   'zircuit',
   'taiko',
   'sei',
+  'swell',
 ];
+
+const MAX_PROTOCOL_FEE = parseEther('100').toString(); // Changing this will redeploy the PROTOCOL_FEE hook
+const CURRENT_ETH_PRICE = 3500; // Changing this will update the existing PROTOCOL_FEE hook
+const protocolFee = (0.5 / CURRENT_ETH_PRICE).toFixed(10).toString(); // ~$0.50 USD
```
Comment (ltyu) [inline typescript/infra/config/environments/mainnet3/warp/configGetters/getRenzoEZETHWarpConfig.ts:34]: sauce: https://t.me/c/2192542084/1467

File: typescript/infra/config/environments/mainnet3/warp/configGetters/getRenzoEZETHWarpConfig.ts [hunk 2]
```diff
@@ -164,6 +173,16 @@ export const ezEthValidators: ChainMap<MultisigConfig> = {
...
     ],
   },
+  swell: {
+    threshold: 1,
+    validators: [
+      {
+        address: '0x7a0f4a8672f603e0c12468551db03f3956d10910', // TODO: NOT READY
+        alias: 'Luganodes',
+      },
+      { address: '0xb6b9b4bd4eb6eb3aef5e9826e7f8b8455947f67c', alias: 'Renzo' },
```
Comment (ltyu) [inline typescript/infra/config/environments/mainnet3/warp/configGetters/getRenzoEZETHWarpConfig.ts:183]: source: https://t.me/c/2192542084/1506

File: typescript/infra/config/environments/mainnet3/warp/configGetters/getRenzoEZETHWarpConfig.ts [hunk 3]
```diff
@@ -23,7 +26,12 @@ const chainsToDeploy = [
   'zircuit',
   'taiko',
   'sei',
+  'swell',
 ];
+
+const MAX_PROTOCOL_FEE = parseEther('100').toString(); // Changing this will redeploy the PROTOCOL_FEE hook
+const CURRENT_ETH_PRICE = 3500; // Changing this will update the existing PROTOCOL_FEE hook
```
Comment (yorhodes) [inline typescript/infra/config/environments/mainnet3/warp/configGetters/getRenzoEZETHWarpConfig.ts:33]: there is a `tokenPrices.json` you can import from

File: typescript/infra/config/environments/mainnet3/warp/configGetters/getRenzoEZETHWarpConfig.ts [hunk 4]
```diff
@@ -41,6 +51,7 @@ const xERC20: Record<(typeof chainsToDeploy)[number], string> = {
   zircuit: '0x2416092f143378750bb29b79eD961ab195CcEea5',
   taiko: '0x2416092f143378750bb29b79eD961ab195CcEea5',
   sei: '0x6DCfbF4729890043DFd34A93A2694E5303BA2703', // redEth
+  swell: '0x2416092f143378750bb29b79eD961ab195CcEea5',
```
Comment (ltyu) [inline typescript/infra/config/environments/mainnet3/warp/configGetters/getRenzoEZETHWarpConfig.ts:54]: source: https://t.me/c/2192542084/1576

File: typescript/infra/config/environments/mainnet3/warp/configGetters/getRenzoEZETHWarpConfig.ts [hunk 5]
```diff
@@ -179,6 +200,7 @@ export const ezEthSafes: Record<string, string> = {
   zircuit: '0x8410927C286A38883BC23721e640F31D3E3E79F8',
   taiko: '0x8410927C286A38883BC23721e640F31D3E3E79F8',
   sei: '0x0e60fd361fF5b90088e1782e6b21A7D177d462C5',
+  swell: '0x435E8c9652Da151292F3981bbf663EBEB6668501',
```
Comment (ltyu) [inline typescript/infra/config/environments/mainnet3/warp/configGetters/getRenzoEZETHWarpConfig.ts:203]: source: https://t.me/c/2192542084/1534

File: typescript/infra/config/environments/mainnet3/warp/configGetters/getRenzoEZETHWarpConfig.ts [hunk 6]
```diff
@@ -164,6 +175,16 @@ export const ezEthValidators: ChainMap<MultisigConfig> = {
       { address: '0x952df7f0cb8611573a53dd7cbf29768871d9f8b0', alias: 'Renzo' },
     ],
   },
+  swell: {
+    threshold: 1,
+    validators: [
+      {
+        address: '0x9eadf9217be22d9878e0e464727a2176d5c69ff8',
```
Comment (ltyu) [inline typescript/infra/config/environments/mainnet3/warp/configGetters/getRenzoEZETHWarpConfig.ts:182]: Screenshot 2024-12-06 at 4 01 19 PM

**Review by yorhodes — APPROVED** [review]

**Review by paulbalaji — COMMENTED** [review]
have you merged the change to registry? need to update the .registryrc 
```
AssertionError: Warp ID EZETH/arbitrum-base-blast-bsc-ethereum-fraxtal-linea-mode-optimism-sei-swell-taiko-zircuit not in registry, the .registryrc or your local registry may be out of date: expected null not to be null
```

**Review by xeno097 — APPROVED** [review]

**Comment by ltyu** [comment]
Closing this. Changes are included in https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/4974

**Pull request closed without merging**